src/JWT/Serializers/Converters/DictionaryStringObjectJsonConverter.cs

```diff
@@ -114,6 +114,16 @@ private static void HandleValue(Utf8JsonWriter writer, string key, object object
                     writer.WriteEndArray();
                     break;
                 }
+                case System.Collections.IEnumerable enumerable:
```

Comment:
It doesn't support something like this:

```c#
case object[] arr:
case System.Collections.IEnumerable arr:
{
    // use arr
}
```
?

Reply:
If no, should we introduce a helper method called form both cases?

Reply:
It doesn't support something like this because the name list conflicts as it is created like a variable:
```
                case object[] list:
                case System.Collections.IEnumerable list:
                {
                    writer.WriteStartArray();
                    foreach (var item in list)
                    {
                        HandleValue(writer, item);
                    }
                    writer.WriteEndArray();
                    break;
                }
```

We can make a helper method instead.

Reply:
Just a quick question: since Array implements IEnunerable, shouldn't having just IEnunerable handle both?

Reply:
Good point! I'll try after work.

Reply:
Sorry for late reply, but you're right! An array implements IEnumerable so I have pushed an even better fix :)